pom.xml

```diff
@@ -2517,11 +2518,18 @@
         <artifactId>undertow-websockets-jsr</artifactId>
         <version>${version.io.undertow.core}</version>
       </dependency>
+
       <dependency>
         <groupId>org.mvel</groupId>
         <artifactId>mvel2</artifactId>
         <version>${version.org.mvel}</version>
       </dependency>
+
+      <dependency>
+        <groupId>javax.xml.bind</groupId>
+        <artifactId>jaxb-api</artifactId>
+        <version>${version.org.jaxb}</version>
+      </dependency>
```

Comment:
In ideal case, if possible, we should be using the JBoss spec jar (which already comes from parent no need to add it here):
```
      <dependency>
        <groupId>org.jboss.spec.javax.xml.bind</groupId>
        <artifactId>jboss-jaxb-api_2.2_spec</artifactId>
      </dependency>
```

Reply:
Otherwise the jaxb spec jar will end up in the classpath twice for someone depending on, for example, this module and optaplanner-persistence-jaxb